firmware/bootloader/src/rusefi_stubs.cpp

```diff
@@ -0,0 +1,29 @@
+#include "main.h"
+#include "hardware.h"
+#include "efiGpio.h"
+#include "global.h"
+
```

Comment:
this file needs an explanation - why do we need stubs? which methods need stubs?